contrib/cirrus/packer/fedora_setup.sh

```diff
@@ -6,130 +6,29 @@
 set -e
 
 # Load in library (copied by packer, before this script was run)
-source /tmp/libpod/$SCRIPT_BASE/lib.sh
+source $(realpath $(dirname ${BASH_SOURCE[0]})/../lib.sh)
 
-req_env_var SCRIPT_BASE PACKER_BUILDER_NAME GOSRC FEDORA_BASE_IMAGE OS_RELEASE_ID OS_RELEASE_VER
+req_env_var SCRIPT_BASE PACKER_BASE INSTALL_AUTOMATION_VERSION PACKER_BUILDER_NAME GOSRC FEDORA_BASE_IMAGE OS_RELEASE_ID OS_RELEASE_VER
 
-install_ooe
-
-export GOPATH="$(mktemp -d)"
-trap "sudo rm -rf $GOPATH" EXIT
-
-$BIGTO ooe.sh sudo dnf update -y
+if [[ $OS_RELEASE_VER -le 31 ]]; then
+    warn "Switching io scheduler to 'deadline' to avoid RHBZ 1767539"
+    warn "aka https://bugzilla.kernel.org/show_bug.cgi?id=205447"
+    echo "mq-deadline" | sudo tee /sys/block/sda/queue/scheduler > /dev/null
```

Comment:
Interesting...

Reply:
sigh...and apparently this is also an issue in F32 https://bugzilla.redhat.com/show_bug.cgi?id=1768092 with a 5.6 kernel :cry:   Fixing...